src/db/models/organization.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+import {sequelize} from './../mysql-connector';
+import * as Sequelize from 'sequelize';
+
+export const Organization = sequelize.define('organizations', {
+    id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+    },
+    name: {
+        type: Sequelize.STRING,
+    },
+    phone: {
+        type: Sequelize.STRING,
+    },
+    email: {
+        type: Sequelize.STRING,
+    },
+    ownerId: {
+        type: Sequelize.STRING,
+    },
+    awsAccountStatus: {
+        type: Sequelize.STRING,
+    },
+    awsClientId: {
+        type: Sequelize.STRING,
+    },
+    awsSecretKey: {
+        type: Sequelize.STRING,
+    },
+    awsAccountKey: {
+        type: Sequelize.STRING,
+    },
+    createdAt: {
+        type: Sequelize.DATE,
+    },
+    updatedAt: {
+        type: Sequelize.DATE,
+    },
+}, {
```

Comment:
I'm accept this one but make sure to use the editor's indentation option before a commit @Rishanthakumar

Reply:
This is auto formatted by the editor itself.